static/usr/lib/systemd/system/modprobe@.service.d/wait-modules.conf

```diff
@@ -0,0 +1,2 @@
+[Unit]
```

Comment:
I do not see `modeprobe@binfmt-misc.service` used in any unit. Is this really used?

Reply:
This is used by some units that have dependencies of they: `Wants=modprobe@<module_name>.service`, like systemd-pstore.service or sys-kernel-config.mount. It is not related to binfmt-misc.